File: sdk/slim/lib/Models/PriceSchedule.php
<?php
/*
 * PriceSchedule
 */
namespace \Models;

/*
 * PriceSchedule
 */
class PriceSchedule {
    /* @var string $iD  */
    private $iD;
/* @var string $name  */
    private $name;
/* @var Bool $applyTax  */
    private $applyTax;
/* @var Bool $applyShipping  */
    private $applyShipping;
/* @var int $maxQuantity  */
    private $maxQuantity;
/* @var Bool $useCumulativeQuantity  */
    private $useCumulativeQuantity;
/* @var Bool $restrictedQuantity  */
    private $restrictedQuantity;
/* @var string $orderType  */
    private $orderType;
/* @var \\Models\ShipmentItem[] $priceBreaks  */
    private $priceBreaks;
/* @var object $xp  */
    private $xp;
}

File: sdk/async-scala/src/main/scala/io/swagger/client/model/PriceSchedule.scala
package io.swagger.client.model

import org.joda.time.DateTime
import java.util.UUID


case class PriceSchedule (
  ID: Option[String],
Name: Option[String],
ApplyTax: Option[Boolean],
ApplyShipping: Option[Boolean],
MaxQuantity: Option[Integer],
UseCumulativeQuantity: Option[Boolean],
RestrictedQuantity: Option[Boolean],
OrderType: Option[String],
PriceBreaks: Option[List[ShipmentItem]],
xp: Option[Any]
)
